pkg/sat/loader_test.go

```diff
@@ -607,30 +607,39 @@ func TestLoader_Load(t *testing.T) {
 			allB = append(allB, newSimplePackage("B", v))
 		}
 
+		atMostOneB := bf.Not(bf.Or(
+			bf.And(x2, x3),
```

Comment:
why is it not sufficient to just say, for example, `bf.Not(bf.Or(x2, x3, x5))` for `eqExpectedAnds`?

Reply:
My motivation was:
1. Each case would need separate condition:
* eq ⇒ `bf.Not(bf.Or(x2, x3, x5))`
* gt ⇒ `bf.Not(bf.Or(x2, x3, x4))`
* lt ⇒ `bf.Not(bf.Or(x4, x5, bf.And(x2, x3)))`
* ge ⇒ `bf.Not(bf.Or(x2, x3, bf.And(x4, x5)))`
* le ⇒ `bf.Not(bf.Or(x5, bf.And(x2, x3), bf.And(x2, x4), bf.and(x3, x4)))`

with *le* getting quite close to the `atMostOneB` condition.
So my argument here is that would make the assertions more complex.

2. The changes in #162 shifted the paradigm of writing these tests from a cryptic, somehow optimized / somehow redundant formulas into more legible intentions, which, I believe `atMostOneB` clearly describes. `atMostOneB` describes the condition we expect to hold in all the cases, regardless the second constraint on a version number.